csharp/src/Ice/IceSSL/TransceiverI.cs

```diff
@@ -314,7 +305,14 @@ public Ice.ConnectionInfo getInfo()
         info.incoming = _incoming;
         info.adapterName = _adapterName;
         info.cipher = _cipher;
-        info.certs = _certs;
+        if (_sslStream is SslStream sslStream && sslStream.RemoteCertificate is X509Certificate2 remoteCertificate)
+        {
+            info.certs = [remoteCertificate];
```

Comment:
The SslStream API doesn't provide the certificate chain. Previously, we saved the chain in the verification callback. However, because we now allow the user to provide the verification callback directly, we will not be able to consistently provide the chain in the info.certs.

I have changed the implementation to only provide the peer certificate, which is the same as what SslStream provides. This will ensure consistent behavior whether the user provides the authentication options or uses the properties-based setup.

Reply:
Sounds fine to me